.github/workflows/release.yaml

```diff
@@ -200,6 +200,48 @@ jobs:
       env:
         GH_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
 
+  build_push_docker_image:
+    needs: [build, release]
+    name: Build and push Docker image
+    permissions:
+      contents: read
+      packages: write # To push the Docker image to ghcr.io
+    uses: ./.github/workflows/_build_docker.yaml
+    with:
+      release-tag: ${{ needs.release.outputs.release-tag }}
+
+  publish_docker_imabe_sbom:
+    name: Release the Docker image sbom
+    needs: [build_push_docker_image, release]
+    permissions:
+      contents: write # To publish the SBOM of the Docker image.
+    runs-on: ubuntu-latest
+    env:
+      SYFT_BIN: ${{ github.workspace }}/bin
+      SBOM_FILE_NAME: macaron-sbom-cyclonedx.json
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      with:
+        fetch-depth: 0
+
+    - name: Install Syft
+      # https://github.com/anchore/syft#installation
+      run: |
+        curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b ${{ env.SYFT_BIN }} v0.74.1
+        ${{ env.SYFT_BIN }}/syft --version
```

Comment:
You can just use `$SYFT_BIN` to read the environment variable.

Reply:
Ah that makes sense. We are running with the shell commands.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/d2fe70dc8bc12b52071d18e885bf63b86f9dcc25